lib/utils/makeChain.js

```diff
@@ -1,6 +1,7 @@
 const {mergeAll} = require('ramda');
+const Raven = require('raven');
 
-const makeChain = (getComposers, data) => {
+const makeChain = Raven.wrap((getComposers, data) => {
```

Comment:
It should work, I was able to send reports constantly. I think it matters where you put Raven.wrap( statement.

Reply:
that's wired,  I tried it on my mac and it does not even get to 'shouldSendCallback' method (Raven creates new domain and run wrapped function and for me domain.on(error) not fired)